lighthouse-treemap/app/src/main.js

```diff
@@ -33,18 +33,15 @@ class TreemapViewer {
    * @param {HTMLElement} el
    */
   constructor(options, el) {
-    const treemapDebugData = /** @type {LH.Audit.Details.DebugData} */ (
+    const scriptTreemapData = /** @type {LH.Audit.Details.TreemapData} */ (
       options.lhr.audits['script-treemap-data'].details);
-    if (!treemapDebugData || !treemapDebugData.treemapData) {
+    if (!scriptTreemapData || !scriptTreemapData.nodes) {
```

Comment:
worth accepting both? I don't suppose we expect many treemap reports to have made it out into the wild thusfar, but it would be nice if they weren't bricked (or maybe file issue to add treemap UI message about compat when an unrecognized payload comes in?) :)

Reply:
> worth accepting both? I don't suppose we expect many treemap reports to have made it out into the wild thusfar

I think it isn't because I think there are none. I shared some but it was as a standalone HTML file.